README.md

```diff
@@ -1,9 +1,346 @@
-OpenMRS Module Appointments Backend
-=================================
+# OpenMRS Module Appointments Backend
+
 This repository acts as the back end for the **Bahmni Appointment Scheduling**.
 
 ## Packaging
-```mvn clean package```
+
+`mvn clean package`
 
 The output is the OMOD file:
-```openmrs-module-appointments/omod/target/appointments-[VERSION].omod```
+`openmrs-module-appointments/omod/target/appointments-[VERSION].omod`
+
+# Setup instructions
+
+This module is incompatiable with the openmrs-appointment-scheduling module, please ensure you remove it either from the admin-page or running this command below from where your modules are located in tomcat directory i.e `/var/lib/OpenMRS/modules` on linux system
+
+```
+sudo rm appointmentscheduling-1.13.0.omod
+```
+
+In order to avoid conflicting tables in the database, please drop the tables below. by running the following sql statements.
+
+```
+set foreign_key_checks=0;
+drop table if exists appointmentscheduling_appointment;
+drop table if exists appointmentscheduling_appointment_block;
+drop table if exists appointmentscheduling_appointment_request;
+drop table if exists appointmentscheduling_appointment_status_history;
+drop table if exists appointmentscheduling_appointment_type;
+drop table if exists appointmentscheduling_block_type_map;
+drop table if exists appointmentscheduling_provider_schedule;
+drop table if exists appointmentscheduling_provider_type_map;
+drop table if exists appointmentscheduling_time_slot;
+set foreign_key_checks=1;
+```
+
+# **Clinical Appointments Scheduling**
+
+# **About**
+
+The appointment scheduling module aims at supporting 3.x users in creating and tracking patient appointments in order to do proper planning and follow up for the services that are being offered in a clinical set-up.
+
+It will also support some other actions; Edit, Cancel and Synchronization of appointments.
+
+# **Where to find it**
```

Comment:
Please separate out the UI onto different documentation. You may want to add a specific section for "Frontends" and within that section, make 3.x widgets one section, so that Bahmni team can also leave pointers and links about how Bahmni's appointment frontend be configured"